Assets/Scripts/Source/Audio/AudioManager.cs

```diff
@@ -32,10 +32,61 @@ public void Play(string sourceName, AudioClip clip, AudioOptions options=null)
         {
             options = AudioOptions.GetDefault();
         }
+        if (clip == null)
```

Comment:
Can this be an overloaded method which does not take a clip as an arg? Or possible call it `Replay`?